.idea/Team-269-Backend.iml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you should add the .idea folder to the gitginore since not every one uses an IDE...
Take a look at this (https://coderwall.com/p/gfjgwg/remove-ide-files-from-your-git-repository-but-keeping-in-your-local-directory) on how you can easily achieve that....

Reply:
Yeah! I noticed after I had made the PR that I should have added the .idea to the .gitignore. Let me fix that! Thanks